publish2rs.py

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env python
+
+import os
+import kinto_http
+import hashlib
+import pathlib
+import uuid
+
+from settings import config
+
+from constants import WEBKIT_LISTS_DIR
+
+def get_config_if_env(env_var, config_section, config_option, fallback=""):
+    """
+    Return the config value if the environment variable exists; otherwise, return the fallback.
+    """
+    if env_var in os.environ:
+        return config.get(config_section, config_option, fallback=fallback)
+    return fallback
```

Comment:
This doesn't look right. Where is `os.environ` read?

```suggestion
    if env_var in os.environ:
        return os.environ[env_var]
    return config.get(config_section, config_option, fallback=fallback)
```